server/chalicelib/s3_alerts.py

```diff
@@ -4,9 +4,13 @@
 
 def routes_for_alert(alert):
     routes = set()
-    for alert_version in alert["alert_versions"]:
-        for informed_entity in alert_version["informed_entity"]:
-            routes.add(informed_entity["route_id"])
+    try:
+        for alert_version in alert["alert_versions"]:
+            for informed_entity in alert_version["informed_entity"]:
+                routes.add(informed_entity["route_id"])
+    except Exception:
+        pass
```

Comment:
Do we want to do some logging here maybe? Something like:
```suggestion
    except Exception as e:
        print("Error fetching alert for route: {e}")
        pass
```
Not sure if this would be useful or not

Reply:
Oh, yeah! :D